core/com/lowagie/text/pdf/PdfPublicKeySecurityHandler.java

```diff
@@ -133,40 +134,40 @@
  * @author Aiken Sam (aikensam@ieee.org)
  */
 public class PdfPublicKeySecurityHandler {
-    
+
     static final int SEED_LENGTH = 20;
-    
+
     private ArrayList recipients = null;
-    
+
     private byte[] seed = new byte[SEED_LENGTH];
 
     public PdfPublicKeySecurityHandler() {
         KeyGenerator key;
         try {
             key = KeyGenerator.getInstance("AES");
             key.init(192, new SecureRandom());
-            SecretKey sk = key.generateKey();            
-            System.arraycopy(sk.getEncoded(), 0, seed, 0, SEED_LENGTH); // create the 20 bytes seed            
+            SecretKey sk = key.generateKey();
+            System.arraycopy(sk.getEncoded(), 0, seed, 0, SEED_LENGTH); // create the 20 bytes seed
```

Comment:
thats a really involved method to geenrate random bytes, but ok its from upstream.